tests/resources/distros/aix72/bin/uname

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
```

Comment:
Any reason for this blank (and the one in file `oslevel` above)?

Reply:
Just for consistency with most of other testing shell scripts :wink:

Reply:
Okay :smile: :man_shrugging: